### PR TITLE
Add roundrobin config for selected sites.

### PR DESCRIPTION
**Pull request by yachang**
This change is [Reviewable](https://reviewable.io/reviews/m-lab/operator/210)

### CODE REVIEW

File: plsync/mlabconfig.py [hunk 1]
```diff
@@ -505,14 +505,26 @@ def export_mlab_site_stats(output, sites):
...
-        sitestats.append({
-            'site': name,
-            'metro': metro,
-            'city': location['city'],
-            'country': location['country'],
-            'latitude': location['latitude'],
-            'longitude': location['longitude']
-        })
+
+        if 'roundrobin' in site.keys():
```
Comment (pboothe) [inline plsync/mlabconfig.py]: Should be 
```python
if site.keys().get('roundrobin', False):
```
That way we won't accidentally turn on Round Robin when passing in `roundrobin=False`

File: plsync/mlabconfig.py [hunk 2]
```diff
@@ -505,14 +505,26 @@ def export_mlab_site_stats(output, sites):
...
+            })
+        else:
+            sitestats.append({
+                'site': name,
+                'metro': metro,
+                'city': location['city'],
+                'country': location['country'],
+                'latitude': location['latitude'],
+                'longitude': location['longitude']
+            })
```
Comment (pboothe) [inline plsync/mlabconfig.py]: Actually, this whole if/else should be replaced with
```python
sitestats.append({
                'site': name,
                'metro': metro,
                'city': location['city'],
                'country': location['country'],
                'latitude': location['latitude'],
                'longitude': location['longitude'],
                'roundrobin': site.keys().get('roundrobin', False)
            })
```

File: plsync/sites.py [hunk 3]
```diff
@@ -121,7 +121,7 @@
     makesite('lax03','173.205.3.64',   '2001:668:1f:1e::',     'Los Angeles_CA', 'US', 33.942500, -118.407200, user_list, count=4, arch='x86_64-r630', nodegroup='MeasurementLabCentos'),
     makesite('lax04','4.15.166.0',     '2001:1900:2100:15::',  'Los Angeles_CA', 'US', 33.942500, -118.407200, user_list, count=4, arch='x86_64-r630', nodegroup='MeasurementLabCentos'),
     makesite('lax05','128.177.109.192','2001:438:fffd:2e::',   'Los Angeles_CA', 'US', 33.942500, -118.407200, user_list, count=4, arch='x86_64-r630', nodegroup='MeasurementLabCentos'),
-    makesite('lba01','109.239.110.0',  '2a00:1a80:1:8::',      'Leeds', 'GB', 53.865800, -1.660560, user_list, nodegroup='MeasurementLabCentos'),
+    makesite('lba01','109.239.110.0',  '2a00:1a80:1:8::',      'Leeds', 'GB', 53.865800, -1.660560, user_list, nodegroup='MeasurementLabCentos', roundrobin=true),
```
Comment (pboothe) [inline plsync/sites.py]: use `True` not `true`

File: plsync/sites.py [hunk 4]
```diff
@@ -179,8 +179,8 @@
...
     makesite('sjc01','70.42.244.64',   '2600:c02:2:82::',      'San Francisco Bay Area_CA', 'US', 37.383300, -122.066700, user_list, count=4, arch='x86_64-r630', nodegroup='MeasurementLabCentos'),  # should be nuq07
     makesite('svg01','81.167.39.0',    '2a01:798:0:13::',      'Stavanger', 'NO', 58.876700, 5.63780, user_list, nodegroup='MeasurementLabCentos'),
-    makesite('syd01','203.5.76.128',   '2001:388:d0::',        'Sydney', 'AU', -33.946100, 151.177000, user_list, nodegroup='MeasurementLabCentos'),
-    makesite('syd02','175.45.79.0',    '2402:7800:0:12::',     'Sydney', 'AU', -33.946100, 151.177000, user_list, count=4, arch='x86_64-r630', nodegroup='MeasurementLabCentos'),
+    makesite('syd01','203.5.76.128',   '2001:388:d0::',        'Sydney', 'AU', -33.946100, 151.177000, user_list, nodegroup='MeasurementLabCentos', roundrobin=true),
+    makesite('syd02','175.45.79.0',    '2402:7800:0:12::',     'Sydney', 'AU', -33.946100, 151.177000, user_list, count=4, arch='x86_64-r630', nodegroup='MeasurementLabCentos', roundrobin=true),
```
Comment (pboothe) [inline plsync/sites.py]: see above

File: plsync/sites.py [hunk 5]
```diff
@@ -193,7 +193,7 @@
     makesite('yvr01','184.105.70.192', '2001:470:1:822::',     'Vancouver','CA', 49.190165, -123.183665, user_list, count=4, arch='x86_64-r630', nodegroup='MeasurementLabCentos'),
     makesite('ywg01','184.105.55.64',  '2001:470:1:81f::',     'Winnipeg', 'CA', 49.905996, -97.237332, user_list, count=4, arch='x86_64-r630', nodegroup='MeasurementLabCentos'),
     makesite('yyc02','65.49.72.192',   '2001:470:1:42c::',     'Calgary', 'CA', 51.1315, -114.0106, user_list, count=4, arch='x86_64-r630', nodegroup='MeasurementLabCentos'),
-    makesite('yyz02','216.66.68.128',  '2001:470:1:70a::',     'Toronto', 'CA', 43.6767, -79.6306, user_list, count=4, arch='x86_64-r630', nodegroup='MeasurementLabCentos'),
+    makesite('yyz02','216.66.68.128',  '2001:470:1:70a::',     'Toronto', 'CA', 43.6767, -79.6306, user_list, count=4, arch='x86_64-r630', nodegroup='MeasurementLabCentos', roundrobin=true),
```
Comment (pboothe) [inline plsync/sites.py]: again

**Comment by yachang** [comment]
Review status: 0 of 2 files reviewed at latest revision, 5 unresolved discussions, some commit checks failed.

---

*[plsync/mlabconfig.py, line 527 at r1](https://beta.reviewable.io/reviews/m-lab/operator/210#-L7yrzjgThqb7xsPHUJe-r1-527:-L7zbQ2G1P5XEzPNDj_j:b-896fix) ([raw file](https://github.com/m-lab/operator/blob/35bc78be30c855450852d0ba55887c7296336e10/plsync/mlabconfig.py#L527)):*
<details><summary><i>Previously, pboothe (Peter Boothe) wrote…</i></summary><blockquote>

Actually, this whole if/else should be replaced with
```python
sitestats.append({
                'site': name,
                'metro': metro,
                'city': location['city'],
                'country': location['country'],
                'latitude': location['latitude'],
                'longitude': location['longitude'],
                'roundrobin': site.keys().get('roundrobin', False)
            })
```
</blockquote></details>

Done.

---

*[plsync/sites.py, line 124 at r1](https://beta.reviewable.io/reviews/m-lab/operator/210#-L7yrzjgThqb7xsPHUJf-r1-124:-L7zbS13DZ6dsKKO0xVT:b-896fix) ([raw file](https://github.com/m-lab/operator/blob/35bc78be30c855450852d0ba55887c7296336e10/plsync/sites.py#L124)):*
<details><summary><i>Previously, pboothe (Peter Boothe) wrote…</i></summary><blockquote>

use `True` not `true`
</blockquote></details>

Done.

---

*[plsync/sites.py, line 183 at r1](https://beta.reviewable.io/reviews/m-lab/operator/210#-L7yrzjgThqb7xsPHUJf-r1-183:-L7zbSNn9i_z_vsds0ta:b-896fix) ([raw file](https://github.com/m-lab/operator/blob/35bc78be30c855450852d0ba55887c7296336e10/plsync/sites.py#L183)):*
<details><summary><i>Previously, pboothe (Peter Boothe) wrote…</i></summary><blockquote>

see above
</blockquote></details>

Done.

---

*[plsync/sites.py, line 196 at r1](https://beta.reviewable.io/reviews/m-lab/operator/210#-L7yrzjgThqb7xsPHUJf-r1-196:-L7zbT-QBUkbM9ky0zFa:b-896fix) ([raw file](https://github.com/m-lab/operator/blob/35bc78be30c855450852d0ba55887c7296336e10/plsync/sites.py#L196)):*
<details><summary><i>Previously, pboothe (Peter Boothe) wrote…</i></summary><blockquote>

again
</blockquote></details>

Done.

---


*Comments from [Reviewable](https://beta.reviewable.io/reviews/m-lab/operator/210)*

**Comment by pboothe** [comment]
:lgtm: , but where do you verify that this information is output correctly?

---

Review status: 0 of 2 files reviewed at latest revision, 5 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/m-lab/operator/210#-:-L7zeIN8lisSflIwpdXG:b4kv0gq)*

**Comment by yachang** [comment]
Yes. I run ./mlabconfig.py --format=sitestats 

and check the output json files.

---

Review status: 0 of 2 files reviewed at latest revision, 5 unresolved discussions, some commit checks failed.

---



*Comments from [Reviewable](https://beta.reviewable.io/reviews/m-lab/operator/210#-:-L7zi5XhEXz_GoWnGTf1:b-v5f09q)*

**Pull request merged**